src/backends/cmake.rs

```diff
@@ -22,6 +22,9 @@ impl Backend for Cmake {
     }
 
     fn build(&self, config: &BuildArgs) -> bool {
+        let create_dir =
+            fs::create_dir_all(format!("{}/build", self.lfc.out.display().to_string())).is_ok();
```

Comment:
Is this to build lfc itself? I think it is a bit misleading to have the lfc variable point to the root of the lingua-franca source directory. I think it should either point to the lfc executable or renamed to lf-root or something.

Reply:
This is the directory where the cmake build files go into. To stick with the current folder structure.

Reply:
Aha, I see.